sympy/core/tests/test_numbers.py

```diff
@@ -1738,9 +1738,9 @@ def test_issue_10020():
 def test_invert_numbers():
     assert S(2).invert(5) == 3
     assert S(2).invert(S(5)/2) == S.Half
-    assert S(2).invert(5.) == 3
+    assert S(2).invert(5.) == 0.5
```

Comment:
This seems to change the meaning invert. The docstring says:
```
    Return the multiplicative inverse of ``self`` mod ``g``
```
Is this consistent with that?

Reply:
When the  mod is not an integer you just get the inverse of the first argument. This is consistent with the behavior tested for Rational (non-Integer) and surds as the 2nd argument.